docs/source/development-guide/style-guide/git-and-github-workflow.rst

```diff
@@ -37,6 +37,9 @@ well.*
    ``imap_processing`` `repository <https://github.com/IMAP-Science-Operations-Center/imap_processing>`_ and clicking
    the ``Fork`` button near the top-right of the page. Follow the various steps to create a fork under your GitHub
    account.
+
+   *Note: Name the fork something general, and not specific to the issue being addressed. The branch title will follow the specific project.*
```

Comment:
For me, I had an option to name my fork. This might just be a rookie git mistake but I named my fork what I should have named my branch. I am actually planning on creating a new fork after these first two pull requests are merged and starting to work from that more appropriately named one. (I don't anticipate people who have used git extensively before to have the same problem, but both Daralynn and I fell down this rabbit hole.)

Reply:
Oh interesting! I hadn't realized that you could name a fork like that.